### ISSUE TITLE
Accessibility Full Conformance test pass for Microsoft Graph Toolkit Playground (A11ytest_tracking, Feature_Task)

### ISSUE THREAD

**Issue by SrujanaAnaganti** [microsoft/tsdoc.org-website#48]

**Comment by SrujanaAnaganti** [comment]
- [ ] 1525:[Accessibility] [Screen reader -Microsoft Graph Toolkit Playground - Overview]: Multiple <H1> headings are defined in the 'Overview' page.
- [ ] 1535:[Accessibility] [Microsoft Graph Toolkit Playground - Navigation LM]: Keyboard focus is landing twice on the 'Microsoft Graph Toolkit playground' link.
- [ ] 1536:[Accessibility] [Screen reader -Microsoft Graph Toolkit Playground - Feedback]: Multiple <H1> headings are defined in the 'Feedback' page. 
- [ ] 1537:[Accessibility] [Microsoft Graph Toolkit Playground - Find components ]: 'Close' button of 'Find components' search field is not accessible.
- [ ] 1538:[Accessibility] [Microsoft Graph Toolkit Playground - Overview]: Keyboard focus is not visible properly on 'Copy' buttons.
- [ ] 1539:[Accessibility] [Microsoft Graph Toolkit Playground - Home page]: Ensures elements with ARIA roles have all required ARIA attributes.
- [ ] 1540:[Accessibility] [Microsoft Graph Toolkit Playground - Home page]: Ensures all ARIA attributes have valid values.
- [ ] 1541:[Accessibility] [Microsoft Graph Toolkit Playground - Overview]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.
- [ ] 1542:[Accessibility] [Microsoft Graph Toolkit Playground - Home page]: Ensures <img> elements have alternate text or a role of none or presentation.
- [ ] 1544:[Accessibility] [Microsoft Graph Toolkit Playground - home page]: Ensures <li> elements are used semantically.
- [ ] 1545:[Accessibility] [Microsoft Graph Toolkit Playground - Overview]: Nested interactive controls are not announced by screen reader
- [ ] 1547:[Accessibility] [Microsoft Graph Toolkit Playground - mgt-people-picker]: Text spacing is not getting applied to the menu items.
- [ ] 1549:[Accessibility] [Microsoft Graph Toolkit Playground]: Keyboard focus is going to non interactive elements through out the URL
- [ ] 1552:[Accessibility] [Microsoft Graph Toolkit Playground]: Few controls are not visible when zoom is set to 200%
- [ ] 1553:[Accessibility] [Microsoft Graph Toolkit Playground]: Navigation LM is getting disappear when zoom is set to 400%. 
- [ ] 1554:[Accessibility] [Microsoft Graph Toolkit Playground - mgt agenda]: +7 button is neither accessible by keyboard nor by mouse
- [ ] 1555:[Accessibility] [Microsoft Graph Toolkit Playground - mgt-agenda]: Keyboard focus is going to background page while navigating in user profile card
- [ ] 1565:[Accessibility] [Microsoft Graph Toolkit Playground-mgt-agenda-Simple]: Controls are not accessible via keyboard in People profile card.
- [ ] 1566: [Accessibility] [Microsoft Graph Toolkit Playground]: Few controls are not visible when high contrast is enabled.
- [ ] 1567:[Accessibility] [Microsoft Graph Toolkit Playground - Sample - General - Login To Show Agenda]: Keyboard focus is not visible on 'Sign in' button.
- [ ] 1569:[Accessibility] [Microsoft Graph Toolkit Playground - Sample - General-Localization]: Multiple controls are not accessible via keyboard in Localization.
- [ ] 1582:[Accessibility] [Microsoft Graph Toolkit Playground - Feedback]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.
- [ ] 1583: [Accessibility] [Microsoft Graph Toolkit Playground - mgt-file-list-Style-Dark Theme]: Ensures every form element has a label.
- [ ] 1584:[Accessibility] [Microsoft Graph Toolkit Playground - mgt-get - Using Image Type]: Ensures buttons have discernible text
- [ ] 1585: [Accessibility] [Microsoft Graph Toolkit Playground - mgt-get -Using Image Type]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.
- [ ] 1586: [Accessibility] [Microsoft Graph Toolkit Playground - mgt-people-picker - Copy Or Cut To Paste]: Ensures ARIA attributes are allowed for an element's role
- [ ] 1631:[Accessibility] [Microsoft Graph Toolkit Playground - mgt-people-picker - People Picker]: Ensures elements with an ARIA role that require parent roles are contained by them.
- [ ] 1632:[Accessibility] [Microsoft Graph Toolkit Playground - mgt-people-picker - People Picker]: Ensures every id attribute value of active elements is unique.
- [ ] 1633:[Accessibility] [Microsoft Graph Toolkit Playground - mgt-people-picker - People Picker]: Ensures <iframe> and <frame> elements have an accessible name. 
- [ ] 1635: [Accessibility] [Screen Reader- Microsoft Graph Toolkit Playground]: Screen reader is announcing link name twice for 'Github' and 'npm'.
- [ ] 1636: [Accessibility] [Screen Reader- Microsoft Graph Toolkit Playground]: Screen reader is not announcing about search result.
- [ ] 1637:[Accessibility] [Screen Reader- Microsoft Graph Toolkit Playground]: Screen reader focus is going all together for left navigation in 'TAB mode.
- [ ] 1638: [Accessibility] [Screen Reader- Microsoft Graph Toolkit Playground]: Screen reader is announcing incorrect state for expand/collapse button
- [ ] 1639:[Accessibility] [Microsoft Graph Toolkit Playground-mgt-people-picker-Dynamic Group Id]: Ensures select element has an accessible name.
- [ ] 1640:[Accessibility] [Microsoft Graph Toolkit Playground-mgt-tasks-Tasks]: Ensures all elements with a role attribute use a valid value
- [ ] 1641:[Accessibility] [Microsoft Graph Toolkit Playground-Sample - General - Theme]: Ensures ARIA attributes are allowed for an element's role.
- [ ] 1642: [Accessibility] [Microsoft Graph Toolkit Playground-Canvas]: Unnecessary keyboard focus is going to 'Canvas' tab
- [ ]
